Docs/source/visualization/visualization.rst

```diff
@@ -33,3 +33,11 @@ files to disk).
 
     sensei
     ascent
+
+If you like the 3D rendering of laser wakefield acceleration
```

Comment:
Just an idea for future docs in `.rst`: I would keep one sentence per line, independent of length.
This makes diffs smaller and saves frustration while writing docs. Also, visibility is not as important here as in code lines.